Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10.8-slim
+LABEL maintainer = "tanyakozakova"
+
+ENV PYTHONUNBUFFERED 1
+ENV API_KEY "d4e3e1249ff546a39b2121545230507"
```

Comment:
don't store your API_KEY here
it should be entered via -e "API_KEY=..." parameter

Reply:
name fixed to docker pull kleotan901/docker-weather-api